rebasehelper/patch_helper.py

```diff
@@ -239,10 +239,34 @@ def _git_rebase(cls):
             patch_dictionary['modified'] = modified_patches
         if inapplicable_patches:
             patch_dictionary['inapplicable'] = inapplicable_patches
+        untouched_patches = cls.get_untouched_patches(deleted_patches, modified_patches, inapplicable_patches)
```

Comment:
Why extra method for this? It could be rewritten for example this way:

```python
patches = [os.path.basename(p.get_path()) for p in cls.patches]
untouched_patches = [p for p in patches if p not in deleted_patches + modified_patches + inapplicable_patches]
```